guides/common/modules/proc_migrating-project-or-smartproxy-using-clone.adoc

```diff
@@ -0,0 +1,24 @@
+[id="migrating-{project-context}-or-proxy-using-clone_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9 by using cloning
```

Comment:
You should probably drop "proxy" from the heading and ID too, right?

Reply:
ack yes. That makes sense.